runtime/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "parity-ethereum-runtime"
```

Comment:
would be better to call it `parity-runtime` IMO because it is not necessarily tied to `parity-ethereum` even though it is only used in `parity-ethereum` at the moment

Reply:
This was my suggestion to make the name long on purpose as I think this crate would be only used for legacy code built on top of or related to parity-ethereum. Ideally, we want to migrate parity-ethereum to async/await and get rid of parity-runtime or rewrite it significantly at least.